pydantic/validators.py

```diff
@@ -1,17 +1,23 @@
 import inspect
+import json
 from collections import OrderedDict
 from datetime import date, datetime, time, timedelta
 from decimal import Decimal, DecimalException
 from enum import Enum
 from pathlib import Path
-from typing import Any
+from typing import Any, Dict, Generic, List, TypeVar
 from uuid import UUID
 
 from . import errors
 from .datetime_parse import parse_date, parse_datetime, parse_duration, parse_time
 from .utils import change_exception, display_as_type
 
 NoneType = type(None)
+JsonObj = TypeVar('JsonObj', Dict[str, Any], List)
+
+
+class Json(Generic[JsonObj]):  # defined not in types.py to avoid circular imports
+    pass
```

Comment:
Move this to `types.py` and implement `get_validators`

Reply:
I also don't htink you need `JsonObj = TypeVar('JsonObj', Dict[str, Any], List)` etc.